cli/azd/pkg/infra/azure_resource_types.go

```diff
@@ -13,6 +13,7 @@ const (
 	AzureResourceTypeStorageAccount          AzureResourceType = "Microsoft.Storage/storageAccounts"
 	AzureResourceTypeKeyVault                AzureResourceType = "Microsoft.KeyVault/vaults"
 	AzureResourceTypeAppConfig               AzureResourceType = "Microsoft.AppConfiguration/configurationStores"
+	AzureResourceTypeAPIM                    AzureResourceType = "Microsoft.ApiManagement/service"
```

Comment:
This should be already added in main.  (Added it to display APIM as created resource).
You might want to rebase and look for `AzureResourceTypeApim` at line 26